docs/driver-access.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Low-level driver access
```

Comment:
Having "Low-level" in the title and subtitle is redundant. I like "Hardware Drivers" for a title.